addons/anonymization/anonymization.py

```diff
@@ -523,7 +523,8 @@ def reverse_anonymize_database(self, cr, uid, ids, context=None):
             data = pickle.loads(base64.decodestring(wizard.file_import))
 
             migration_fix_obj = self.pool.get('ir.model.fields.anonymization.migration.fix')
-            fix_ids = migration_fix_obj.search(cr, uid, [('target_version', '=', '8.0')])
+            import openerp.release
```

Comment:
don't make an import in the middle of the code, move this to top `from openerp.release import version_info`